docs/how-to/ci.rst

```diff
@@ -0,0 +1,160 @@
+Building your App in CI with GitHub Actions
+===========================================
+
+This GitHub `Actions <https://docs.github.com/en/actions>`__ workflow provides
+the basic framework necessary to test, build, and package a Briefcase project
+in CI for Windows, Linux, macOS, iOS, and Android.
+
+The same set of steps are used for each platform via a `matrix strategy
+<https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/running-variations-of-jobs-in-a-workflow>`__.
+This will create a separate job for each target platform with specific
+configuration.
+
+The workflow steps:
+
+* Check out the current GitHub repository
+
+  * This will be the version of your repository that triggered the CI run
+  * So, for a pull request (PR), this is the code being submitted in the PR
+  * After the PR is merged, this it is the code in your default branch
+
+* Install Python 3.12
```

Comment:
Should probably clarify that *this* example workflow uses 3.12, but you can use any support Python version.